var/spack/repos/builtin/packages/openexr/package.py

```diff
@@ -64,12 +67,20 @@ class Openexr(CMakePackage):
 
     variant("debug", default=False, description="Builds a debug version of the libraries")
 
-    depends_on("cmake@3.12:", when="@3:", type="build")
+    depends_on("cmake@3.12:", when="@2.4:", type="build")
     depends_on("pkgconfig", when="@:2", type="build")
     depends_on("imath", when="@3:")
     depends_on("ilmbase", when="@:2")
     depends_on("zlib")
 
+    # fix build of 2.5 with GCC 13
+    patch(
+        "https://github.com/AcademySoftwareFoundation/openexr/pull/1499/commits/8bb802a4ea1c9628e6b77a4bfa9e6ec9bb97ca05.patch?full_index=1",
+        sha256="5919bfeb3f87acc455373a774d6b24a6db6c089f09e6ff1821d46cd1491dbdbf",
+        when="@2.5",
+    )
+    conflicts("@:2.4 %gcc@13")
```

Comment:
```suggestion
    conflicts("@:2.4 %gcc@13:")
```
Presumably newer versions will have the same issue?